README.md

```diff
@@ -55,7 +55,7 @@ SPIFFE is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
 * [CONTRIBUTING](/CONTRIBUTING.md)
 * [GOVERNANCE](/GOVERNANCE.md)
 
-### SIGs & Working Groups<a name="sigs"></a>
+### SIGs & Working Groups
```

Comment:
I'm guessing the anchor was in place to allow urls with a fragment `#sigs`. Since those URLs might still be in the wild, I don't see a reason to break them.

Reply:
Fixed